### ISSUE TITLE
Usernames & profiles

### ISSUE THREAD

**Issue by psokolhessner** [startyourlab/conventions#3]
**Starting problem**
Use of profile features, choice of profile picture, standardization (or lack thereof) of usernames can create uncertainty and inconsistency within a collaborative environment. 

**Possible solution**
State basic standards for effective & appropriate usernames (based on actual name? first initial full last name?), what parts of profiles to complete in what way (i.e. which contact info should be included? Actual picture of the person for profile pic? Full name?), while pointing out where flexibility is fine.

**Comment by aridyckovsky** [comment]
Great idea for enhancement @psokolhessner. I assume this is for one or both of Slack and GitHub conventions?

- Slack is probably the most straightforward with display name vs full name, can also include reasons to include 'extras' like phone number, email, skype. 
- GitHub will have a straightforward level (profile picture, username, display name) and then a more advanced level (pinning repositories, including a profile description, etc). For the username on GitHub, we'll want to make sure people understand that this is also their unique identifier on GitHub -- that username is really, really important because it shows up for everything you contribute to, and even is in the URL to your profile.

Do those distinctions seem reasonable at per-tool level?

**Comment by psokolhessner** [comment]
Yes, agreed @aridyckovsky. This issue seems perhaps _only_ important for new users/people unfamiliar with the platform. The advanced features of such profiles will become apparent with time, so only a nod to them feels necessary.

**Comment by aridyckovsky** [comment]
This also seems like it would be a good thing to reference **and expand on** within a lab manual repository template. Most likely this would be within an "onboarding" or "new lab member" section. I think it's worth having separation -- in this repo, focus on the _what_ of the conventions with minimal _how_ and _why_, regardless of new or advanced user types. In the lab manual, give more of the _how_ and _why_ for new users.

**Comment by psokolhessner** [comment]
Agreed. I would suggest a mostly minimal section (here), with perhaps the biggest point being "consider giving a touch of guidance on this, esp. if any of following things are important to you (e.g. having contact info for all members in the workspace) or you are already defining related standards elsewhere (e.g. professionalism in conduct & presentation)."
